native/src/main/java/io/ballerina/stdlib/grpc/protobuf/utils/BalFileGenerationUtils.java

```diff
@@ -47,7 +48,9 @@ private BalFileGenerationUtils() {
      * @param command protoc executor command.
      * @throws CodeGeneratorException if an error occurred when executing protoc command.
```

Comment:
update the API Docs with return value

Reply:
check other places

Reply:
Updated in [673fbdf](https://github.com/ballerina-platform/module-ballerina-grpc/pull/739/commits/673fbdf3937245c3406b7258b82810dfce3b25f6)